SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetConnection.cpp

```diff
@@ -22,6 +22,8 @@ DECLARE_CYCLE_STAT(TEXT("UpdateLevelVisibility"), STAT_SpatialNetConnectionUpdat
 
 USpatialNetConnection::USpatialNetConnection(const FObjectInitializer& ObjectInitializer)
 	: Super(ObjectInitializer)
+	, bReliableSpatialConnection(false)
```

Comment:
drive by?

Reply:
yep - not necessary but i'd rather be explicit about default initialization